/**/camera_node:
  ros__parameters:
    video_capture: '/dev/video0'
    video_resolution_width: 640
    video_resolution_height: 480
    video_framerate: 20 #sec

/**/imu_node:
  ros__parameters:
    acc_x_offset: -0.165931074 
    acc_y_offset: 0.201548310205276
    acc_z_offset: -0.141471371
    gyro_x_offset: 1.5990274809160256
    gyro_y_offset: 0.4370078625954107
    gyro_z_offset: -0.196853435114507
    acc_x_low_cutoff: 0.1
    acc_y_low_cutoff: 0.05
    acc_z_low_cutoff: 0.5
    gyro_x_low_cutoff: 5.0
    gyro_y_low_cutoff: 1.0
    gyro_z_low_cutoff: 5.0

/**/ultrasonic_node:
  ros__parameters:
    tr_pin: 11
    ec_pin: 8
